### DISCUSSION TITLE
For how many instances do you plan to use CTVBot?

### DISCUSSION THREAD

**Discussion by jlplenio** [KevinBytesTheDust/CVAmp#173] [Polls]
By adding more and more functionality, the CPU utilization per instance increases. This reduces the maximum number of instances that can be run on a given system. So it would be very informative to know what the normal utilization of CTVBot is. Are you trying to reach the maximum number of instances, or are you satisfied with a modest number?

**Comment by MRSSSMR** [comment]
I also added 64GB of RAM to use more than 100 of them, but it's getting heavier after 0.5.1 version, so it's a shame that the BOT is decreasing
I suggest if it is difficult to divide into a version with many functions and a light version

**Reply by jlplenio** [reply]
Hey @MRSSSMR. That is a good idea, like spawning of light-weight instances.
